android/src/main/java/com/svtek/reactnative/RNAndroidSharerModule.java

```diff
@@ -80,10 +110,12 @@ private void shareImageViaFacebook(String filePath, final Promise promise) {
 
   private void shareVideoViaFacebook(String filePath, final Promise promise) {
     try {
-      ShareVideo video = new ShareVideo.Builder()
-              .setLocalUrl(this.uriFromFilePath(filePath)).build();
+      ShareVideo video = new ShareVideo.Builder().setLocalUrl(this.uriFromFilePath(filePath)).build();
+
```

Comment:
Please remove this extra new line as these three variables are closely related.

Reply:
Removed.